src/current/molt/migrate-failback.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Migration Failback
+summary: Learn how to fail back from a CockroachDB cluster to a PostgreSQL or MySQL database.
+toc: true
+docs_area: migrate
+---
+
+Failback can be performed after you have loaded data into CockroachDB and are replicating ongoing changes. Failing back to the source database ensures that—in case you need to roll back the migration—data remains consistent on the source.
+
+{% assign tab_names_html = "Load and replicate;Phased migration;Failback" %}
+{% assign html_page_filenames = "migrate-to-cockroachdb.html;migrate-in-phases.html;migrate-failback.html" %}
+
+{% include filter-tabs.md tab_names=tab_names_html page_filenames=html_page_filenames page_folder="molt" %}
+
+## Before you begin
+
+- [Enable rangefeeds]({% link {{ site.current_cloud_version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) in the CockroachDB SQL shell:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING kv.rangefeed.enabled = true;
+    ~~~
+
+- Ensure that your CockroachDB deployment has a valid [Enterprise license]({% link {{ site.current_cloud_version }}/licensing-faqs.md %}).
```

Comment:
Should we show the sample command of how to do this?
```
SET CLUSTER SETTING cluster.organization = 'org';
SET CLUSTER SETTING enterprise.license = 'license';
```

Reply:
Is this requirement specific to MOLT or just to using CRDB in general? If the latter, I might prefer to exclude the step entirely since it's not specifically about MOLT -- unless customers often get stuck here.

Reply:
Im with @taroface on this one. I don't think we need to include CRDB related set up in molt things. Besides explicit cluster settings for the kv.rangefeed.enabled. But we should assume customers using this stuff have an entierprise license and org already set up for their cluster.